docs/external-contributions/external-contributions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: external-contributions
+title: External Contributions
+hide_title: true
+---
+
+# External Contributions
+
+Contributions to Cumulus may be made in the form of [PRs to the repositories directly](https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md) or through externally developed tasks and components. Cumulus is designed as an ecosystem that leverages Terraform deployments and AWS Step Functions to easily integrate external components.
+
+This list may not be exhaustive and represents components are open source and owned externally that have been tested with the Cumulus system. For more information and contributing guidelines, visit the respective GitHub repositories.
```

Comment:
```suggestion
This list may not be exhaustive and represents components that are open source and owned externally and that have been tested with the Cumulus system. For more information and contributing guidelines, visit the respective GitHub repositories.
```

Just a thought: do we list all Cumulus repositories anywhere? Should we?

Reply:
I don't know that we do. They are tagged. I need to go through these docs and find a place to do that.